frameworks/PHP/webman/benchmark_config.json

```diff
@@ -2,8 +2,27 @@
   "framework": "webman",
   "tests": [{
```

Comment:
New feature:
If you add your github user to the maintainers line, you will be pinged with any new change.

```
{
  "framework": "webman",
  "maintainers": ["walkor"],
  "tests": [{
  ...

```

You cand do the same with Workerman.